manifests/params.pp

```diff
@@ -36,6 +36,11 @@
   $rdeck_home = '/var/lib/rundeck'
   $service_logs_dir = '/var/log/rundeck'
 
+  $rdeck_uuid = $::serialnumber ? {
+    undef   => fqdn_uuid($::fqdn),
+    default => $::serialnumber,
```

Comment:
Entirely my fault, but for consistency/readability, we should probably use the $facts hash for the `default` case too (even though we don't strictly need to).